hello-world-aar/build.gradle

```diff
@@ -10,4 +10,7 @@ dependencies {
     testCompile "junit:junit:${project.property('junit.version')}"
 }
 
-apply from: 'https://raw.githubusercontent.com/chrisdoyle/gradle-fury/master/gradle/android-support.gradle'
+//use this line for your projects
+//apply from: 'https://raw.githubusercontent.com/chrisdoyle/gradle-fury/master/gradle/android-support.gradle'
+
+apply from: "${rootDir}/gradle/android-support.gradle"
```

Comment:
Now that I'm looking at this, I'm wondering if we should provide alternative instructions for including the files locally as you have done here, or to include it using the URL-based approach.  What do you think?  Definitely lends itself to "offline" building a little better.